### ISSUE TITLE
Can't use Tabs in another Tabs

### ISSUE THREAD

**Issue by alexisinwork** [reactjs/react-tabs#54]
```javascript
<Tabs selectedIndex={1}>
              <TabList>
                <Tab>Disclosure Req.</Tab>
                <Tab>Page Instructions</Tab>
              </TabList>
              <TabPanel>
                <Instructions sectionName="DisclosuresRequirements" source="disclosures/instruction" />
              </TabPanel>
              <TabPanel>
                <Tabs selectedIndex={1}>
                  <TabList>
                    <Tab>Disclosure Req.</Tab>
                    <Tab>Page Instructions</Tab>
                 </TabList>
                  <TabPanel>
                    <Instructions sectionName="DisclosuresRequirements" source="disclosures/instruction" />
                  </TabPanel>
                  <TabPanel>
                    <Instructions sectionName="EffortSummary" source="effort/instruction" />
                  </TabPanel>
                </Tabs>
              </TabPanel>
            </Tabs>
```
When I click on inner second Tab it change parent second tab.
Who can help me with this problem?

**Comment by strafe89** [comment]
I faced same problem.

**Comment by KyleAMathews** [comment]
Just ran into this too

**Comment by mzabriskie** [comment]
Sorry I don't have a lot of time right now to look into this. If someone wants to take a stab at a fix I'd welcome a PR.

**Comment by strafe89** [comment]
It happens because onlick (select tab) event is attached on whole component including tab panels. And when you put another tabs inside tab panel body -  it fires twice. 
I bypassed this issue by attaching one more onclick listener - on parent tab panel - using stopPropagation().

**Comment by jamesaevans** [comment]
@strafe89 could you elaborate on your solution?

**Comment by kirankashalkar** [comment]
Can one of the merge requests be accepted that solve the issue?

**Comment by kirankashalkar** [comment]
Please accept a merge. Nested tabs is an important feature to make this component production ready.

**Comment by peternelson815** [comment]
Will this be fixed/merged soon? I like this component but need nesting - I'll have to use something else if this isn't resolved.

**Comment by kirankashalkar** [comment]
@peternelson815 - what did you end up going with?

**Comment by danez** [comment]
Thanks for reporting, this will be fixed in the next release

**Comment by SirCmpwn** [comment]
I'm running 0.5.5 and still encountering this bug.

**Comment by danez** [comment]
It is not yet released, it will be in the next release.

**Comment by SirCmpwn** [comment]
Ah, okay. Thanks.

**Comment by kirankashalkar** [comment]
When is the next version planned to be released?

**Comment by danez** [comment]
No release planed. This was released in 0.6 I think

**Comment by howardwu1** [comment]
> It happens because onlick (select tab) event is attached on whole component including tab panels. And when you put another tabs inside tab panel body - it fires twice. I bypassed this issue by attaching one more onclick listener - on parent tab panel - using stopPropagation().

Thank you! I was using two sets of tabs that weren't related by an expected nesting (having a documentation with navigation tabs showing off various pages, one with a set of nested tabs within that doc page, so the nesting was independent of the documentations tabs).
